src/controllers/rental.controller.ts

```diff
@@ -0,0 +1,41 @@
+import express from 'express';
+import { inject, injectable } from 'inversify';
+import {
+  controller,
+  httpPost,
+  interfaces,
+  next,
+  request,
+  response,
+  httpPut
+} from 'inversify-express-utils';
+import { IRentalService } from '../application/services/irental.service';
+import { TYPES } from '../di/types';
+import { Request } from '../domain/Request';
+
+@controller('/api/v1/rental')
+export class RentalController implements interfaces.Controller {
+  constructor(
+    @inject(TYPES.IRentalService) private rentalService: IRentalService
+  ) {}
+
+  @httpPost('/request')
+  private async arrive(@request() req: express.Request, @response() res: express.Response) {
```

Comment:
Be consequent, modify method name to request instead of arrive ? This is because your route name is request

Reply:
fixed